.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,7 @@
+<!--
+
+If you are a [PowerShell Team](https://github.com/orgs/PowerShell/people) member, please make sure you choose the Reviewer(s) and Assignee for your PR.
+If you are not, feel free to choose them if you are familiar with the team. Or leave them blank and let maintainers choose them for you.
```

Comment:
`If you are not a team member`

Replace `Or leave` with `Alternatively, `

After `Or` Replace the first `Them` with `the fields`.